### PR TITLE
Added Projects-page

### PR DESCRIPTION
**Pull request by InbarDanieli**
adding page for mentor's projects

### CODE REVIEW

File: pages/projects/vest.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import Navbar from '../../components/layout/navbar/navbar';
+import style from '../../styles/projects.module.scss';
+
+export default function vast() {
+  return (
+    <div>
```
Comment (MichalPorag) [inline pages/projects/vest.tsx:6]: please change to fregment

File: pages/projects/vest.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+      <Navbar />
+      <div className={style.container}>
+        <h3 className={style.title}>Vest</h3>
+        <div className={style.star}>
```
Comment (MichalPorag) [inline pages/projects/vest.tsx:10]: please use before/ after (on the number of stars element) to create the star element

File: pages/projects/vest.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+          <div className={style.starText}>30K</div>
```
Comment (MichalPorag) [inline pages/projects/vest.tsx:11]: please move it to be a child of the title

File: pages/projects/vest.tsx [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import Navbar from '../../components/layout/navbar/navbar';
+import style from '../../styles/projects.module.scss';
+
+export default function vast() {
+  return (
+    <div>
+      <Navbar />
+      <div className={style.container}>
+        <h3 className={style.title}>Vest</h3>
```
Comment (MichalPorag) [inline pages/projects/vest.tsx:9]: please use flexbox to center the title

File: styles/projects.module.scss [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+  color: #ce5a55;
+}
+
+.star {
+  width: 51px;
+  height: 51px;
+  transform: rotate(15deg);
+  float: right;
+  margin-left: 90px;
+  background-image: url("../public/Star\ 1.svg");
```
Comment (MichalPorag) [inline styles/projects.module.scss:26]: please try to use shortcuts

File: styles/projects.module.scss [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+  display: flex;
+  justify-content: center;
+}
+
+.title {
+  margin-top: 25px;
+  position: absolute;
+  width: 1424px;
+  height: 58px;
+  font-style: normal;
```
Comment (MichalPorag) [inline styles/projects.module.scss:12]: please remove no needed styles

File: pages/projects/vest.tsx (hunk 1 above)
Comment (MichalPorag) [inline pages/projects/vest.tsx:6]: please move project page to a component that we can use later in more cases

**Review by MichalPorag — CHANGES_REQUESTED** [review]
Great job! you did quite a lot in one day! just a few changes, please look at the comments  :)

File: styles/projects.module.scss (hunk 5 above)
Comment (InbarDanieli) [reply styles/projects.module.scss:14]: what do you mean by shortcuts?

**Review by MichalPorag — APPROVED** [review]

**Pull request merged**